spec/integ/matrix-client-syncing.spec.ts

```diff
@@ -194,6 +195,46 @@ describe("MatrixClient syncing", () => {
             expect(fires).toBe(3);
         });
 
+        it("should emit ClientEvent.Room when invited", async () => {
```

Comment:
This test isn't exercising the code path, probably due to it not having IDB, maybe we need to import `import 'fake-indexeddb/auto';`

Reply:
You can ignore the strict errors stuff, its a bit faulty at excluding things you haven't touched